scripts/commands/gotonpc.lua

```diff
@@ -0,0 +1,25 @@
+---------------------------------------------------------------------------------------------------
+-- func: gotonpc
+-- auth: Chronos77
+-- desc: Goes to the target player.
+---------------------------------------------------------------------------------------------------
+
+cmdprops =
+{
+    permission = 1,
+    parameters = "s"
+};
+
+function onTrigger(player, target)
+    if (target == nil) then
+        player:PrintToPlayer("You must enter a valid npc id.");

```

Comment:
If I have to know the ID to use this, its not so usefull..The spawn info is litterly right there when you look up the NPC in the db. Its also in the header of almost every npc script in conveniant copy and past form with (at)pos.
https://github.com/DarkstarProject/darkstar/blob/master/scripts/zones/Bastok_Markets/npcs/Brygid.lua#L7
Should actually be in all NPC's headers but some got forgotten.

Your print after this implies its by name not ID, but the code before that is clearly expecting an ID with the exception of the comand info paramters expectinga  string instead of an integer.

Have you actually tried to use this command in game?